sdk/schemaregistry/client/src/main/java/com/azure/schemaregistry/client/SchemaRegistryClient.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+package com.azure.schemaregistry.client;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.function.Function;
+
+/**
+ *  Interface that defines operation for registering and fetching schemas and schema information to and from a
+ *  schema registry store.
+ */
+public interface SchemaRegistryClient {
```

Comment:
Clients have to be concrete classes instantiated from `ClientBuilders`. See [guidelines](https://azure.github.io/azure-sdk/java_design.html#service-clients) on creating service clients. Also, add the necessary annotations.